src/main/webapp/data-visualization.js

```diff
@@ -131,13 +131,13 @@ function createDataArray(censusDataArray, isCountyQuery) {
   censusDataArray = censusDataArray.slice(1); // get rid of header row
```

Comment:
Maybe I'm looking at an outdated version, but I think the bug might still be here -- when you reassign censusDataArray to censusDataArray.slice(1), you get rid of the header, so the reference to censusDataArray[0] in line 134 will actually be censusDataArray[1]. An adjustment that you could make is define another variable, censusArrayWithoutHeader = censusDataArray.slice(1) and then traverse that, so that when you reference censusDataArray later it is still the original data.

Reply:
Yeah, you're definitely right that the header is still getting erased. But I realized that actually, it makes a lot of sense to look at the first row and not the header row. The header can't tell us what's a percentage and what's not, but the numbers themselves can.